src/doc/html.md

```diff
@@ -30,9 +30,20 @@ attribute to `<html>` as in this example:
 <html class="no-js" lang="en">
 ```
 
+
+## Document Title
```

Comment:
This makes me wonder how far our documentation should go and at what point it starts to look like an intro to HTML. This looks like general knowledge which would better fit in a getting started with web development tutorial then in the H5BP docs.

On the other hand, it's important to emphasise  things like this. What does the rest think?